tools/reviewer/job/sync/github_api.proto

```diff
@@ -29,37 +31,37 @@ message File {
 message PullRequestFilesRequest {
```

Comment:
Please add URLs for the GitHub API also here, similar to how you did in the static finals.

Reply:
Done.